commit 4a90933b843370f461433d7ec71ad0538e5302b9
Author: Robert Mustacchi <rm@joyent.com>
Date:   2019-05-09T21:55:10+00:00 (5 months ago)
    
    OS-7787 make clean fails if ./tools/builder is already clean
    Reviewed by: Josh Wilsdon <jwilsdon@joyent.com>
    Approved by: Josh Wilsdon <jwilsdon@joyent.com>

diff --git a/tools/builder/Makefile b/tools/builder/Makefile
index 292d62eb..4b9bce31 100644
--- a/tools/builder/Makefile
+++ b/tools/builder/Makefile
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2010 Joyent Inc., All rights reserved.
+# Copyright 2019 Joyent, Inc.
 #
 
 CC=gcc
@@ -21,4 +21,4 @@ users.c:
 	./build_users_c.sh $(SRC_ROOT)/overlay/generic > users.c.tmp && mv users.c.tmp users.c
 
 clean:
-	rm $(TARGETS) $(OBJS)
+	rm -f $(TARGETS) $(OBJS)
